### ISSUE TITLE
Allow configuration of SSL certificate verification

### ISSUE THREAD

**Issue by amorphic** [RidersDiscountCom/HypChat#37]
Requests provides the [verify](http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification) kwarg to allow a user to:
* __True:__ verify the host's certificate using Requests' internal certificate bundle
* __False:__ do not verify the host's certificate
* __/path/to/cert/bundle.pem:__ verify the host's certificate using a local certificate bundle file

The last two options are requried when using self-signed certificates, i.e. when using SSL inside an organisation.

HypChat does not currently support the `verify` argument:

    class HypChat(object):
        def __init__(self, token, endpoint='https://api.hipchat.com'):
            self._requests = _requests(auth=BearerAuth(token))

A workaround exists for using a local certificate bundle file: export the `REQUESTS_CA_BUNDLE` environment variable. But it would be great if HypChat explicitly supported the `verify` kwarg if provided by the user.

**Comment by philipsd6** [comment]
I agree, this should be explicitly exposed, but at least you can work around it:
```python
hipchat._requests.verify = False
```

**Comment by successamp** [comment]
I'm happy to accept pull requests on this.
